pkg/domain/infra/abi/images.go

```diff
@@ -608,15 +606,6 @@ func (ir *ImageEngine) Sign(ctx context.Context, names []string, options entitie
 			return nil, errors.Errorf("cannot determine canonical Docker reference for destination %s", transports.ImageName(rawSource.Reference()))
 		}
 
-		// create the signstore file
```

Comment:
Do we not need to create the signstore file?

Reply:
Move the comments down to L635 where the sigstore is created. Deleted lines of code are not used  since the repo and digest can be got from image reference.